perses/rjmc/geometry.py

```diff
@@ -351,6 +357,9 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         neglected_angle_terms : list of ints
             list of indices corresponding to the angle terms in the corresponding system that are neglected (i.e. which are to be
             placed into the lambda perturbation scheme)
+        omitted_growth_terms : dict
```

Comment:
`geometry_engine` returns omitted valence and 1,4 terms which can be generated in ring-closure procedures

Reply:
1) it's `omitted_terms` above
2) can we just put neglected_angle_terms in this dict?